action.yml

```diff
@@ -2,60 +2,51 @@
 
 name: Run MATLAB Tests
 description: >-
-  Run all tests in a MATLAB project and generate test artifacts
+  Run MATLAB and Simulink tests and generate artifacts
 inputs:
   source-folder:
     description: >-
-      Source code folder Location of the folder containing source code, relative
-      to the project root folder. The specified folder and its subfolders are
-      added to the top of the MATLAB search path. To generate a code coverage
-      report, MATLAB uses only the source code in the specified folder and its
-      subfolders. You can specify multiple folders using a colon-separated or a
-      semicolon-separated list.
+      Source code folder location of the folder containing source code
```

Comment:
Location of the folder containing source code

Reply:
Or

Location of the folder containing source code, relative to the project root folder